docs/runtime/minimal-runtime.md

```diff
@@ -136,20 +146,20 @@ impl sdk::Runtime for Runtime {
                     // Alice.
```

Comment:
All of this could probably be simplified using:
```rust
BTreeMap::from([
    (sdk::testing::keys::alice::address(), BTreeMap::from([(Denomination::NATIVE, 1_000_000_000])),
    (sdk::testing::keys::bob::address(), /* ... */),
    /* ... */
]);